pkg/limayaml/limayaml.go

```diff
@@ -319,3 +320,18 @@ type CACertificates struct {
 	Files          []string `yaml:"files,omitempty" json:"files,omitempty" jsonschema:"nullable"`
 	Certs          []string `yaml:"certs,omitempty" json:"certs,omitempty" jsonschema:"nullable"`
 }
+
+// Engine is the name of the container engine, e.g. "docker", "containerd", that contains a list of sockets to monitor.
+type Engine struct {
+	Sockets []string `yaml:"sockets,omitempty" json:"sockets,omitempty"`
+}
+
+type Kubernetes struct {
+	Configs []string `yaml:"configs,omitempty" json:"configs,omitempty"` // Paths to Kubernetes config files, e.g. "/etc/rancher/k3s/k3s.yaml"
+}
+
+type PortMonitor struct {
+	Docker     Engine     `yaml:"docker,omitempty" json:"docker,omitempty"`
+	Containerd Engine     `yaml:"containerd,omitempty" json:"containerd,omitempty"`
+	Kubernetes Kubernetes `yaml:"kubernetes,omitempty" json:"kubernetes,omitempty"`
+}
```

Comment:
All the new settings must be included in `default.yaml` with comments explaining the default settings:

```yaml
portMonitors:
  docker:
    # ...
    sockets: []
  ...
```